skflow/__init__.py

```diff
@@ -117,6 +117,15 @@ def predict(self, X):
             return pred
         return pred.argmax(axis=1)
 
+    def predict_proba(self, X):
```

Comment:
Thanks for merging, currently observing in Chile [i'm an astronomer] so had little time to implement the changes you suggested. I hope to be able to contribute more in the future.